community-operators/cluster-logging/cluster-logging.v0.0.1.clusterserviceversion.yaml

```diff
@@ -62,8 +63,10 @@ spec:
   description: |
     The Cluster Logging Operator for OKD provides a means for configuring and managing your aggregated logging stack.
 
+    It is recommended to deploy this operator to the **openshift-logging** namespace.
+
     Once installed, the Cluster Logging Operator provides the following features:
-    * **Create/Destroy**: Launch and create an aggregated logging stack in the `openshift-logging` namespace.
+    * **Create/Destroy**: Launch and create an aggregated logging stack to support the entire OKD cluster.
```

Comment:
If the operator only supports SingleNamespace, how is it supposed to support "the entire OKD cluster"? Unless the application pod is responsible for that.

Reply:
The logging stack supports the entire OKD cluster.  The cluster-logging-operator only runs in a single namespace.